internal/v1/handler_blueprints_test.go

```diff
@@ -0,0 +1,46 @@
+package v1
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+	"testing"
+
+	"github.com/osbuild/image-builder/internal/tutils"
+)
+
+// note: this scenario needs to talk to a simulated osbuild-composer API
+func TestInsertBlueprint(t *testing.T) {
+	id := uuid.New()
+	versionId := uuid.New()
+
+	dbase, err := dbc.NewDB()
+	require.NoError(t, err)
+
+	db_srv, tokenSrv := startServerWithCustomDB(t, "", "", dbase, "../../distributions", "")
+	defer func() {
+		err := db_srv.Shutdown(context.Background())
+		require.NoError(t, err)
+	}()
+	defer tokenSrv.Close()
+
+	var result CreateBlueprintResponse
+
+	blueprintName := "Blueprint Name"
+	description := "description of the blueprint"
+
+	err = dbase.InsertBlueprint(id, versionId, "000000", "500000", blueprintName, description, json.RawMessage("{}"))
+	require.NoError(t, err)
+
+	blueprintEntry, err := dbase.GetBlueprint(id, "000000", "500000")
+	require.NoError(t, err)
+
+	respStatusCode, body := tutils.GetResponseBody(t, fmt.Sprintf("http://localhost:8086/api/image-builder/v1/experimental/blueprint/%s", blueprintEntry.Id), &tutils.AuthString0)
+	require.Equal(t, 200, respStatusCode)
+	err = json.Unmarshal([]byte(body), &result)
+	require.NoError(t, err)
+	require.Equal(t, blueprintEntry.Name, blueprintName)
+	require.Equal(t, blueprintEntry.Description, description)
+}
```

Comment:
This test does not test the UpdateBlueprint function. Is this some missing test?